packages/api/src/account/queries/get_account_archive.sql

```diff
@@ -0,0 +1,15 @@
+SELECT
+  account_archive.account_archiveid AS "accountArchiveId",
+  account_archive.accountid AS "accountId",
+  account_archive.accessrole AS "accessRole",
+  account_archive.type,
+  account_archive.status
+FROM
+  account_archive
+INNER JOIN
+  account
+  ON account_archive.accountid = account.accountid
+WHERE
+  account_archive.archiveid = :archiveId
+  AND account.primaryemail = :email
+  AND account_archive.status = 'status.generic.ok';
```

Comment:
Note to self: this probably makes sense, but I wonder if we want to use this for declining an archive invitation? I should investigate how that works right now.

Reply:
(Relevant because that would be in status "pending")